core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/ui/experiments-header/experiments-ui-header.component.scss

```diff
@@ -0,0 +1,19 @@
+:host {
```

Comment:
use colors from `variables` file

Reply:
removed components, it is not part of this PR.